### DISCUSSION TITLE
<attr-name> syntax

### DISCUSSION THREAD

**Discussion by Semigradsky** [csstree/csstree#149] [Q&A] [answered]
In your tests I see that `[ng:cloak]` selector is valid. Also in https://csstree.github.io/docs/validator.html code is shown as valid:
```css
[ng:cloak] {
  color: red
}
```

But in https://csstree.github.io/docs/syntax/#Type:attr-name `ng:cloak` value is shown as invalid.

As I see `<ident-token>` can't include `colon` symbol.

UPD: @lahmatiy is this non-standard behaviour in `csstree`?
As I see there is no `':'` in diagram https://drafts.csswg.org/css-syntax/#ident-token-diagram

**Comment by lahmatiy** [comment, marked as answer]
Yep, you're right, that's non-standard behaviour which inherited from CSSO tests. I guess it was once important for Angular. However, I believe that no reason to support it now and it should be removed.

**Comment by lahmatiy** [comment]
Removed in [2.0.4](https://github.com/csstree/csstree/releases/tag/v2.0.4)
